Tests/kaas/kaas-sonobuoy-go-example-e2e-framework/README.md

```diff
@@ -0,0 +1,86 @@
+# SCS kaas Sonobuoy testsuite and K8s e2e-test-framework
+
+This directory holds all conformance tests that aim to validate the scs kaas infrastructure
+
+## Sonobuoy plugin description
+
+> NOTE: There is a Sonobuoy blog post walking through this plugin, its benefits, and how to use it. See: <https://sonobuoy.io/plugin-starter/>
+
```

Comment:
The beginning of this readme should be changed to reflect that this is just an example.

Reply:
Agreed, I integrated a hint that this is an example.